rosidl_generator_cpp/rosidl_generator_cpp/__init__.py

```diff
@@ -174,31 +204,40 @@ def primitive_value_to_cpp(type_, value):
         'char',
         'int8', 'uint8',
         'int16', 'uint16',
-        'int32', 'uint32',
-        'int64', 'uint64',
+        'float64'
     ]:
         return str(value)
 
-    if type_.type in ['float32']:
-        return '%sf' % value
-
-    if type_.type in ['float64']:
+    if type_.type == 'int32':
         return '%sl' % value
 
+    if type_.type == 'uint32':
+        return '%sul' % value
+
+    if type_.type == 'int64':
+        return '%sll' % value
+
+    if type_.type == 'uint64':
+        return '%sull' % value
+
+    if type_.type == 'float32':
+        return '%sf' % value
+
     if type_.type == 'string':
         return '"%s"' % escape_string(value)
 
+    # raise ValueError("unknown primitive type '%s'" % type_.type)
```

Comment:
I'd remove this commented out code; we already have a TODO at the top of the function.

Reply:
yeah that's a remnant I started raising but figured we needed a more thorough approach than just raise in a handful of places so I added the TODO on top and commented out my previous docblocks and raise. I'll remove them